### DISCUSSION TITLE
must react-native-papaer use react-native-vector-icons ?

### DISCUSSION THREAD

**Discussion by sacru2red** [callstack/react-native-paper#4007] [Polls]
react-native-paper support web (rnw) projects

[use react-native-vector-icons's MaterialCommunityIcons](https://callstack.github.io/react-native-paper/docs/guides/react-native-web#load-the-material-community-icons)

but font file is so big, heavy (300kb~900kb)

we can use `@iconify/react`
[iconify](https://pictogrammers.com/docs/guides/iconify/)
like
```
import { Icon } from '@iconify/react'
// https://pictogrammers.com/library/mdi/icon/arrow-down-bold-box/
<Icon icon="mdi:arrow-down-bold-box" fontSize={30} />
```

if remove react-native-vector-icons in project
user's cannot use src/components/MaterialCommunityIcon

**Comment by RainPlays09** [comment]
Use `@expo/vector-icons` instead
